BUILDING.md

```diff
@@ -12,8 +12,127 @@ For building the manual, see the README in the [Manual repository](https://githu
 
 Windows
 -------
+ Visual studio version:
+ 1.) Download:
+   https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017
+
+   Install: Tools for Visual Studio 2019
+     MSVC v142 
+     Windows 10 SDK
+     C++ Cmake Tools for Windows
+     MSVC v141 - VS2017 C++ x64/x86 build Tools
+
+ 2.) Download:
+   http://download.qt.io/official_releases/qt/5.9/5.9.7/qt-opensource-windows-x86-5.9.7.exe
+   Install 
+      MSVC 2017 64-bit
+ 3.) Compile (Console)
+
+   set Qt5_DIR=E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\lib\cmake\Qt5
+   cd {DwarfTherapist Source Dir}
+   mkdir build
+   cd bild
+   cmake -G "Visual Studio 15 2017 Win64" ..\
```

Comment:
Doesn't cmake need the environment setup to configure msvc? I use `call "%VS140COMNTOOLS%VsDevCmd.bat"` (for MSVC 2015), I don't see anything equivalent in your instructions.

Reply:
After installing VS you get various links in the start menu for opening a console window with the environment you want to build. You can select the platform by choosing the correct console window. But I should add this to the guide to make it complete.

Reply:
It is was not clear in the instructions. You need to explain how to open the correct console. Reading "Console", I would have opened cmd.exe.

The VsDevCmd.bat script is good for batch scripts, but the special console may be better for interactive command line.

Reply:
Explained the console stuff more precisely in the current version of my guide. Hopefully less confusion now.